libkbfs/folder_block_ops.go

```diff
@@ -160,6 +160,16 @@ func (fbo *folderBlockOps) GetState(lState *lockState) overallBlockState {
 	return dirtyState
 }
 
+func (fbo *folderBlockOps) getBlockFromCache(ptr BlockPointer,
```

Comment:
I disagree that this should be inlined; I like that it's a separate method. However, it should be named to more explicitly communicate that it's checking both caches.